core/src/it/scala/org/ensime/core/RefactoringHandlerSpec.scala

```diff
@@ -7,20 +7,24 @@ import java.nio.charset.Charset
 import java.text.SimpleDateFormat
 import java.util.Date
 
+import scala.concurrent.ExecutionContext.Implicits.global
+
 import org.ensime.api._
 import org.ensime.fixture._
 import org.ensime.util.EnsimeSpec
 import org.scalatest.Assertions
-import scala.concurrent.ExecutionContext.Implicits.global
 
 class RefactoringHandlerSpec extends EnsimeSpec
     with IsolatedAnalyzerFixture
     with RichPresentationCompilerTestUtils
     with RefactoringHandlerTestUtils {
 
   val encoding = "UTF-16"
+  val ensimeProject = EnsimeConfigFixture.EmptyTestProject.projects.head.copy(
+    scalacOptions = List("-encoding", encoding)
+  )
   def original = EnsimeConfigFixture.EmptyTestProject.copy(
```

Comment:
this could be a bit cleaner. e.g.

```scala
def original = {
  val empty = EnsimeConfigFixture.EmptyTestProject
  empty.copy(empty.projects.head.copy(... ))
}
```

you could even use a shapeless lens, if you wanted to avoid the nesting of `copy` calls. If we depended on Monocle it'd be even cleaner (but we can't).